geolib/models/dsheetpiling/internal.py

```diff
@@ -24,6 +22,7 @@
     DSerieVersion,
 )
 from geolib.utils import make_newline_validator
+from pydantic import confloat, conint, conlist, constr
```

Comment:
Why did you move external packages a few lines further down the code?

Reply:
This is done automatically by isort, which for some reason was not really working before but it should be an automatic step for everyone.
In my case it gets run by VSCode each time I save a file locally.